### DISCUSSION TITLE
Tailwind CSS only very partly usable

### DISCUSSION THREAD

**Discussion by ModdyLP** [nunocoracao/blowfish#536] [General]
Hay,

you mention, in the README, that tailwind CSS is used for design. But if I try to use this, I often get weird behaviors in reason of missing tailwind classes. e.g. padding is defined as p-[1-10] and in the compiled main.css in the assets there are only p-0/1/4/5/6. The rest are missing. 

Why does that differ ? If you compiled tailwind CSS, there should be all standard CSS classes available.

Kind Regards

**Comment by nunocoracao** [comment]
The framework only compiles what is actually used. Please get into the theme folder and run 'npm dev' while  developing and 'npm build' for final compile. Might need to install npm dependencies using npm install.

**Reply by ModdyLP** [reply]
oh yeah, that might help. Thank you, I found for me a quicker solution. But I remember this for the future development.
